doc/apiary/v2/fiware-ngsi-simple-cookbook.apib

```diff
@@ -0,0 +1,872 @@
+FORMAT: 1A
+HOST: http://fiware-ngsi2.apiblueprint.org/
+
+# FIWARE NGSI v2 Cookbook
+
+<!--
+ Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+
+ This file is part of Orion Context Broker.
+
+ Orion Context Broker is free software: you can redistribute it and/or
+ modify it under the terms of the GNU Affero General Public License as
+ published by the Free Software Foundation, either version 3 of the
+ License, or (at your option) any later version.
+
+ Orion Context Broker is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+ General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+
+ For those usages not covered by this license please contact with
+ iot_support at tid dot es
+-->
+
+FIWARE-NGSI Simple API (v2) Cookbook.
+
+This is a cookbook with recipes that show how to use FIWARE-NGSI v2 aka FIWARE-NGSI Simple API.
+Use this examples for learning and enjoy!
+
+This is a work in progress and is changing on a daily basis.
+Please send your comments to fiware-ngsi@lists.fiware.org
+
+# FIWARE-NGSI-Simple API Root [/v2]
+
+This resource allows you to obtain a list of subresources published by the API:
+
+* entities to deal with your NGSI entities
+* types to deal with your entity types
+* subscriptions to deal with subscriptions to context changes
+* registrations to deal with context data by registering external providers
+
+## Retrieve the Entry Point [GET]
+
++ Response 200 (application/json)
+
+        {
+            "entities_url":      "/v2/entities",
+            "types_url":         "/v2/types",
+            "subscriptions_url": "/v2/subscriptions",
+            "registrations_url": "/v2/registrations"
+        }
+
+## Listing context entities [/v2/entities{?type,id,idPattern,limit,offset}]
```

Comment:
Same comments than in previous PR about the relationship of the parameters in `{ ... }` at `##` level and `###` subsections:

> Not sure of the parameters part has to be filled... Maybe it should be the union set of the required/optional parameters for all HTTP verbs (GET, POST, PUT, etc.) on the /v2/entities URL path?